grails-app/services/org/pih/warehouse/core/DocumentTemplateService.groovy

```diff
@@ -184,15 +201,26 @@ class DocumentTemplateService {
         metadata.addFieldAsList("orderItems.totalPrice")
         metadata.addFieldAsList("orderItems.expectedShippingDate")
 
+        metadata.addFieldAsList("groupedOrderItems.code")
```

Comment:
The naming is fine for now. 

But just want to mention that if we end up supporting different grouping methods then we'll need to come up with a convention to distinguish between these different representations. Coming up with a convention now would prevent us from having to refactor a bunch of document templates in the future.

orderItemsByCategory
orderItemsByProductCode
orderItemsByProductCodeAndUom

Reply:
I don't love variable names like `orderItemsBy...` but just wanted to point out that we're painting ourselves into a corner with `groupedOrderItems`